src/Testcontainers/Builders/CommonDirectoryPath.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace DotNet.Testcontainers.Builders
+{
+  using System.IO;
+  using System.Text.RegularExpressions;
+
+  public readonly struct CommonDirectoryPath
+  {
+    /// <summary>
+    /// Sets the DirectoryPath to the parent directory path of the bin directory.
+    /// </summary>
+    public static readonly CommonDirectoryPath BuildRoot = new CommonDirectoryPath(GetParentBinPath());
+
+    private CommonDirectoryPath(string directoryPath)
+    {
+      this.DirectoryPath = directoryPath;
+    }
+
+    public string DirectoryPath { get; }
+
+    private static string GetParentBinPath()
```

Comment:
This will work for this projects as well, as it just what ever the parent directory of bin is, if they want to copy the dockerfile to the default output (bin) then  they can use the original method.

Reply:
You're right. I had something that bound the path to a `build` directory in mind... Sorry.